docs/reference/resource-compute/list-of-commands-available-on-a-compute-resource-provisioned-by-juju/list-of-juju-introspect-macros/juju_presence_report.md

```diff
@@ -1,4 +1,47 @@
 (juju_presence_report)=
 # `juju_presence_report`
 
-TBA
+The presence report shows the status of juju agent connections to a controller
+and can be used to establish a view on which agents are "alive". A connection
+is registered whenever an agent logs in to the api server on the machine on
+which the report is generated.
+
+The agents for each model are listed together beneath the UUID of that model.
+
+## Usage
+Must be run on a juju controller machine.
+```code
+juju_presence_report
+```
+
+## Example output
```

Comment:
I love this example output section in these introspection command docs. I wish we had something similar for the `juju` CLI commands, at least the ones with a rich output where it might be helpful to guide the user through the output a little bit.